diff --git a/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_source/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDdlParser.java b/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_target/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDdlParser.java
index add493ebe..7ee9d21e7 100644
--- a/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_source/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDdlParser.java
+++ b/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_target/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDdlParser.java
@@ -834,6 +834,15 @@ public class MySqlDdlParser extends DdlParser {
             column.charsetName("utf8");
         }
 
+        if (Types.DECIMAL == dataType.jdbcType()) {
+            if (dataType.length() == -1) {
+                column.length(10);
+            }
+            if (dataType.scale() == -1) {
+                column.scale(0);
+            }
+        }
+
         if (tokens.canConsume("CHARSET") || tokens.canConsume("CHARACTER", "SET")) {
             String charsetName = tokens.consume();
             if (!"DEFAULT".equalsIgnoreCase(charsetName)) {
