diff --git a/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_source/src/main/java/org/jsapar/compose/bean/BeanComposer.java b/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_target/src/main/java/org/jsapar/compose/bean/BeanComposer.java
index 93b03ad67..cef34f6b2 100644
--- a/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_source/src/main/java/org/jsapar/compose/bean/BeanComposer.java
+++ b/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_target/src/main/java/org/jsapar/compose/bean/BeanComposer.java
@@ -136,7 +136,7 @@ public class BeanComposer<T> implements Composer, BeanEventListener<T>, ErrorEve
 
         for (Cell cell : line) {
             String sName = cell.getName();
-            if (sName == null || sName.isEmpty())
+            if (sName == null || sName.isEmpty() || cell.isEmpty())
                 continue;
 
             try {
diff --git a/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_source/src/main/java/org/jsapar/parse/bean/Bean2Cell.java b/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_target/src/main/java/org/jsapar/parse/bean/Bean2Cell.java
index 5a9b4edd4..bdecb9b26 100644
--- a/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_source/src/main/java/org/jsapar/parse/bean/Bean2Cell.java
+++ b/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_target/src/main/java/org/jsapar/parse/bean/Bean2Cell.java
@@ -228,6 +228,9 @@ public class Bean2Cell {
 
     public void assign(Object bean, Cell cell)
             throws BeanComposeException, InvocationTargetException, IllegalAccessException, InstantiationException {
+        if(cell.isEmpty()) {
+            return;
+        }
         if (isLeaf()) {
             assignProperty(bean, cell);
             return;
