diff --git a/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_source/src/main/java/org/springframework/data/repository/util/QueryExecutionConverters.java b/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_target/src/main/java/org/springframework/data/repository/util/QueryExecutionConverters.java
index 4cb24fdf9..5dcb0f054 100644
--- a/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_source/src/main/java/org/springframework/data/repository/util/QueryExecutionConverters.java
+++ b/home/sahand/Desktop/CC4APCA/CC4APCA/tmp/checkouts/.diff_target/src/main/java/org/springframework/data/repository/util/QueryExecutionConverters.java
@@ -525,7 +525,7 @@ public abstract class QueryExecutionConverters {
 		 */
 		@Override
 		protected Object wrap(Object source) {
-			return ReflectionUtils.invokeMethod(OF_METHOD, source);
+			return ReflectionUtils.invokeMethod(OF_METHOD, null, source);
 		}
 
 		public static WrapperType getWrapperType() {
